### ISSUE TITLE
Addd script to turn off screen when computer go to sleep mode

### ISSUE THREAD

**Issue by ZILosoft** [Plippo/screenpad-tools#2]
Thank you to create that module and script!
Please add stcript to automatic turn off screen when computer go to sleep mode

**Comment by Plippo** [comment]
Unfortunately I can't reproduce this problem as for me, the screen already turns off when going to sleep without needing any scripts or modifications. If it doesn't work for you, you can try putting a script to /usr/lib/pm-utils/sleep.d/ (or the corresponding location for your system) that just calls "screenpad 0" when the notebook is suspended.

**Comment by s-light** [comment]
@ZILosoft in the past i had problems with sleepmode in general- in this case the computer did not go correclty to sleepmode or did not wakeup - so maybe it is the same in your case?

i also can confirm that for me the screen correclty is shut off by the system itself when it goes to sleep mode -no additional things needed.

**Comment by boscowitch** [comment]
I don't think I have the problem with sleep mode, because then power is correclty cut.

But I somehow habe a similar problem.
If gnome switches off the Screen to save power before going to sleep mode, it can only switch off the main one and the second one just gets blacked and is still glowing....

 this is not sleep mode,more like screen switch of at inactivity or if you lock the screen on purpose with win + L.
gnome/gdm also switches the main monitor off but the second is just blacked.

maybe this is what the original author confused with sleep mode ...

**Comment by s-light** [comment]
> gets blacked and is still glowing....

yeah that is a think... i think the screen can be switched off - but in this case the screen completely switches off - and vanishes from the system - there for windows get moved to the main screen and other *dual-screen-setup-things* get broken....

for that iam vary happy that it *only gets dark* - 
in this moment its bright sunshine outside - but if i remember correctly - we changed so that the lowest brightness value really is 0 and this only turns off the backlight - so that would be the *leave the screen active but shutoff the backlight*  option.
maybe its possible to add a script that gets triggered on this *screen sleep* event that also handles the main screen and in this set the second-screen-brightness to `0`..
